pages/common/git-pull.md

```diff
@@ -0,0 +1,11 @@
+# git pull
+
+> Fetch branch from a remote repo and merge it to current repo
```

Comment:
Maybe "local repo" instead of "current repo", to further drive the point home? And how about not using jargon (repository instead of repo)? Do you think that's being too cautious?